functionary/builder/utils.py

```diff
@@ -303,11 +304,27 @@ def _create_functions_from_definition(definitions, package: Package):
         function_obj.schema = _generate_function_schema(
             name, function_def.get("parameters")
         )
+        function_obj.active = True
         db_functions.append(function_obj)
 
     return db_functions
 
 
+def _deactivate_functions(definitions, package: Package):
+    """Grabs all functions from DB, deactivates any that aren't in the definition"""
+    db_functions = Function.objects.all()
+    for function_def in definitions:
+        name = function_def.get("name")
+        print(name)
+        try:
+            func = Function.objects.get(package=package, name=name)
+            db_functions.remove(func)
+        except Function.DoesNotExist:
+            pass
+    for inactive_functions in db_functions:
```

Comment:
Another minor quibble: you've named the variable `inactive_functions`, but it is 1) a singular function, not multiple, and 2) not actually inactive at this point in time.  I'd just call it `function`.

Reply:
Fixed to function